sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -648,3 +648,8 @@
 .formio-component-file ul li.list-group-item {
   border: 1px solid rgba(0, 0, 0, 0.125) !important;
 }
+
+.banner-header {
```

Comment:
For the header in banner in form.io
![image](https://user-images.githubusercontent.com/54600590/212430713-e363a1b7-01e4-4225-90d7-2bfe7b1e0f0e.png)

Reply:
Is this change supposed to affect all the banners? For instance, should it not be changed for all `alert-warning`, `alert-info`, and `alert-danger`?